docs/analyzer/user_guide.md

```diff
@@ -905,16 +905,15 @@ usage: CodeChecker analyze [-h] [-j JOBS]
                            [-e checker/group/profile]
                            [-d checker/group/profile] [--enable-all]
                            [--verbose {info,debug,debug_analyzer}]
-                           logfile
+                           input
 
 Use the previously created JSON Compilation Database to perform an analysis on
 the project, outputting analysis results in a machine-readable format.
 
 positional arguments:
-  logfile               Path to the JSON compilation command database files
-                        which were created during the build. The analyzers
-                        will check only the files registered in these build
-                        databases.
+  input                 The input of the analysis can be either a compilation
+                        database JSON file, a path to a source file or a path
+                        to a directory containing source files.
```

Comment:
Collected some more places where analyze input is mentioned:
It is at your discretion how much you edit.
- README.md 
  - 71, 116
- analyzer/user_guide.md
  -859, 910
- usage.md
  -125
The numbers are line numbers where the new mention of the new option is missing.